packages/plugin-infra/src/web3-token-types.ts

```diff
@@ -0,0 +1,87 @@
+export enum EthereumTokenType {
+    Native = 0,
+    ERC20 = 1,
+    ERC721 = 2,
+    ERC1155 = 3,
+}
+
+// #region ERC721
+export interface ERC721Token {
+    type: EthereumTokenType.ERC721
+    address: string
+    chainId: number
+}
+
+export interface ERC721ContractDetailed extends ERC721Token {
+    name: string
+    symbol: string
+    baseURI?: string
+    iconURL?: string
+}
+
+export interface ERC721TokenInfo {
+    name?: string
+    description?: string
+    tokenURI?: string
+    mediaUrl?: string
+    imageURL?: string
+    owner?: string
+    // loading tokenURI
+    hasTokenDetailed?: boolean
+}
+
+export interface ERC721TokenDetailed {
```

Comment:
```suggestion
export interface DetailedERC721Token {
```
I see ERC20Token**S**Detailed in the codebase, it's hard to distinguish from ERC20TokenDetailed, besides according to English grammar, I think `detailed` should be put in front of `token`. ( `detailed token` or `token in detail`)

Reply:
OK，`DetailedNonFungibleToken`.